documentation/teaching/grading.md

```diff
@@ -6,7 +6,7 @@ If you run the test under the "Grader" config, you can run just the grading scri
 This assumes that you are running inside the gradescope docker container.
 
 ```
-sbt "Grader / testOnly dinocpu.LabXGrader"
+sbt "Grader / testOnly dinocpu.test.grader.LabXGrader"
```

Comment:
This is fine for now... we might want to try to find a way to make this easier for students.